tests/integration/pip_parse/WORKSPACE

```diff
@@ -7,6 +7,13 @@ load("@rules_python//python:repositories.bzl", "py_repositories", "python_regist
 
 py_repositories()
 
+# Needed for rules_cc 0.2.10
+load("@bazel_features//:deps.bzl", "bazel_features_deps")
```

Comment:
Instead of copy/pasting this same setup to every downstream workspace, create a second setup function and load and call that.

`python/repositories_deps.bzl`

which does the 4 lines